docs/_articles/en/v3.x/scanner-commands/dfa.md

```diff
@@ -38,7 +38,7 @@ Important: If your codebase is complex, increase the Java heap space to avoid Ou
       Writes output to a file.
 
   -p, --projectdir=_projectdir_
-      (required) Provides the relative or absolute root project directory used to set the context for Graph Engine's analysis. Project directory must be a path, not a glob. Specify multiple values as a comma-separated list.
+      Provides the relative or absolute root project directory used to set the context for Graph Engine's analysis. Project directory must be a path, not a glob. Specify multiple values as a comma-separated list.
```

Comment:
Should you match here what we have the in cli. This is what you approved for the cli a few weeks ago:

Provides the relative or absolute root project directory used to set the context for Graph Engine's analysis. Project directory must be a path, not a glob. If --projectdir isn’t specified, a default value is calculated. The default value is a directory that contains all the target files.

Note that `Specify multiple values as a comma-separated list.` definitely is incorrect.

Lastly, this should also be on the run.md page as well since this option is relevant to run if --engine sfge is specified.